schemas/com.snowplowanalytics.snowplow.storage.rdbshredder/shredding_complete/jsonschema/1-0-0

```diff
@@ -0,0 +1,86 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for information about the shredded data, for loader to read",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.storage.rdbshredder",
+    "name": "shredding_complete",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "base": {
+      "description": "Blob storage path to the root of the batch",
+      "type": "string",
+      "maxLength": 1024
```

Comment:
We could add a check that it starts with `s3://`, `s3a://` or `s3n://`

Reply:
I think that `format: uri` is what we need here. At some point RDB Loader will be working with GCS and Snowflake.